src/core/public/chrome/ui/header/recent_items.tsx

```diff
@@ -0,0 +1,111 @@
+/*
+ * Copyright OpenSearch Contributors
+ * SPDX-License-Identifier: Apache-2.0
+ */
+import React, { useMemo, useState } from 'react';
```

Comment:
Added a new line above
```suggestion

import React, { useMemo, useState } from 'react';
```